client/src/pages/docs/plugins.mdx

```diff
@@ -90,10 +90,10 @@ docker run -dit -p 8000:8000 --name plugins_container plugins_image
 docker ps
 ```
 
-`docker ps` should return the following, if everything worked.  You should then be able to run the client and set your .env file as discussed above.
+`docker ps` should return the following, if everything worked. You should then be able to run the client and set your .env file as discussed above.
 
-CONTAINER ID   IMAGE           COMMAND                  CREATED          STATUS          PORTS                    NAMES
-12ed73ede6a0   plugins_image   "uvicorn --host 0.0.…"   21 seconds ago   Up 19 seconds   0.0.0.0:8000->8000/tcp   plugins_container
+CONTAINER ID IMAGE COMMAND CREATED STATUS PORTS NAMES
```

Comment:
In MD this displays weirdly, worth doing it as a table or something for clarity?

Reply:
Makes Sense I just used prettier to format the files and it did this formatting

Reply:
Actually this should probably be a code block

Reply:
Fixed